test.sh

```diff
@@ -1,8 +1,12 @@
 #!/bin/sh
 
 ./ganache.sh &>/dev/null &
+GANACHE_PID=$!
```

Comment:
I tried this too.  $! returns the PID of the ganache.sh script, not the node process actually running ganache.

Reply:
`pkill -P $GANACHE_PID` below will take care of it. What it does is it finds the child PID (the node process) and kills it. It works on my mac but also I expect `pkill` to behave similarly on other Linux shells.